example/flake.nix

```diff
@@ -88,8 +88,11 @@
           };
 
         # nix run .#ponysay up to start the process
+        # nun run .#ponysay attach to show the output
+        # nix run .#ponysay down to stop the process
         packages.ponysay = (import inputs.process-compose-flake.lib { inherit pkgs; }).makeProcessCompose {
           modules = [{
+            cli.up.detached = true;
```

Comment:
Why detached?

Reply:
That's the whole point of all my contributions, to get detached mode running, so process-compose-flake can be used like "docker compose up -d" :sweat_smile: